### PR TITLE
Add wait for ClusterServiceBroker func in fast-integration test

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- add waitForClusterServiceBroker() and run it before trying to provision Service Instance in skr-svcat-migration-test

**Related issue(s)**

### CODE REVIEW

File: tests/fast-integration/utils/index.js [hunk 1]
```diff
@@ -491,6 +491,18 @@ function waitForSubscription(name, namespace = "default", timeout = 180000) {
   );
 }
 
+function waitForClusterServiceBroker(name, timeout = 90000) {
+  return waitForK8sObject(
+      `/apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers`,
+      {},
+      (_type, _apiObj, watchObj) => {
+        return watchObj.object.metadata.name.includes(name);
```
Comment (ksputo) [inline tests/fast-integration/utils/index.js:499]: please also add condition for ready state:
```golang
        watchObj.object.status.conditions.some(
          (c) => c.type == "Ready" && c.status == "True"
        )
        ```

Comment (szwedm) [reply tests/fast-integration/utils/index.js]: added

**Review by ksputo — APPROVED** [review]

**Review by rakesh-garimella — APPROVED** [review]

**Pull request merged**